libavfilter/dnn/dnn_backend_tf.c

```diff
@@ -200,13 +201,8 @@ static DNNReturnType set_input_output_tf(void *model, DNNData *input, const char
 
     sess_opts = TF_NewSessionOptions();
     // protobuf data for auto memory gpu_options.allow_growth=True
-    // gpu_options.visible_device_list="gpuid" , default: 0
-    uint8_t config[7] = { 0x32, 0x5, 0x20, 0x1, 0x2a, 0x01, 0x30 };
-    if(tf_model->gpuid > 0x30) {
-        config[6] = (uint8_t)tf_model->gpuid;
-    }
-    TF_SetConfig(sess_opts, (void*)config, 7, tf_model->status);
-
+    uint8_t config[4] = { 0x32, 0x02, 0x20, 0x1 };
```

Comment:
What do this represent?

Reply:
That's  gpu option protobuffer for setting allow_growth in Tensorflow.  This option is absolutely necessary  to save GPU memory.